includes/plugins/class-woocommerce-memberships.php

```diff
@@ -79,6 +79,9 @@ public static function filter_lists( $lists ) {
 		if ( ! is_array( $lists ) || empty( $lists ) ) {
 			return $lists;
 		}
+		if ( ! function_exists( 'wc_memberships_is_post_content_restricted' ) ) {
```

Comment:
Hmm so apparently this happens if woo memberships is active but woocommerce is not.

I think it's best to fix the check on the `init` method, that checks `class_exists( 'WC_Memberships_Loader' )`. If Woo and/or Woo memberships is not active, the hooks should not even be initalized

Reply:
Good idea, added in acacf58eba286b3325a5a0b5f3b07a7e4345d5d4